_layouts/base.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+
+{% include head.html %}
+
+<body>
+    <div class="ui vertical masthead aligned container">
+        {% include header.html %}
+
+        {{ content }}
+    </div>
+
+    {% include try-geo.html %}
+
+</body>
+
+{% include footer.html %}
+
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/fetch/1.0.0/fetch.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.4.0/highlight.min.js"></script>
+<script>hljs.initHighlightingOnLoad();</script>
+<script src="{{ site.baseurl }}dist/geo-map.js"></script>
+<script src="{{ site.baseurl }}dist/clipboard.min.js"></script>
```

Comment:
Don't use `dist` as a folder for JS. It doesn't make sense in a static website generator: everything will output into the `_site` folder, which is your intellectual “dist”. Use `js`.